docs/root/configuration/http_conn_man/stats.rst

```diff
@@ -13,7 +13,6 @@ statistics:
    downstream_cx_total, Counter, Total connections
    downstream_cx_ssl_total, Counter, Total TLS connections
    downstream_cx_http1_total, Counter, Total HTTP/1.1 connections
-   downstream_cx_websocket_total, Counter, Total WebSocket connections
```

Comment:
Do we have stats on new style upgrade/websocket? I'm wondering if we are losing observability w/ the new flow?

Reply:
Added in https://github.com/envoyproxy/envoy/pull/4767